spec/association_spec.rb

```diff
@@ -570,6 +570,38 @@ class Tag < ActiveRecord::Base ; end
     end
   end
 
+  context 'manually defined through association' do
+    before(:each) do
+      create_tables(
+          "posts", {}, {},
+          "tags", {}, {},
+          "post_tags", {:id => false}, { :post_id => { foreign_key: true}, :tag_id => { foreign_key: true}}
```

Comment:
Nit: the spacing inside {} is inconsistent.

Reply:
it was matching how it was from where I copied it.. but changed this anyways to a less contrived example.